src/components/get-side-chain-pnk-button.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styled from "styled-components/macro";
+import { Link } from "react-router-dom";
+import { Button } from "../adapters/antd";
+import { counterPartyChainIdMap, isSupportedSideChain, useTokenBridgeApi } from "../api/token-bridge";
+import TokenSymbol from "./token-symbol";
+
+export default function GetSideChainPnkButton() {
```

Comment:
Just a UX suggestion here. Could we do here something similar to what we do on token-bridge-card.js:74? Asking Metamask to switch chain, instead of displaying the "you are on the wrong network" error and asking the user to switch manually.

Reply:
Since this component is only displayed when users are already on xDAI/Sokol, doing that to go back to Mainnet/Kovan isn't possible, because the the call doesn't work with the default networks.

Reply:
I see 🤔 Thanks!

Reply:
See this for more details:
https://docs.metamask.io/guide/rpc-api.html#wallet-addethereumchain

Reply:
>MetaMask stringently validates the parameters for this method, and will reject the request if any parameter is incorrectly formatted. In addition, MetaMask will reject the request under the following circumstances:
>
>   - If the RPC endpoint doesn't respond to RPC calls.
>   - If the RPC endpoint returns a different chain ID when eth_chainId is called.
>   - **If the chain ID corresponds to any default MetaMask chains.**